src/screens/PickList/index.tsx

```diff
@@ -16,11 +16,31 @@ import { Card }  from 'react-native-paper';
 import SCAN from '../../assets/images/scan.jpg';
 import TICK from '../../assets/images/tick.png';
 import CLEAR from '../../assets/images/icon_clear.png';
+import Radio from '../../components/Radio';
+import DropDown from 'react-native-paper-dropdown';
+
+// TODO: Refactor (pull from api, when shortage reason codes will be available)
+const SHORTAGE_REASON_CODES = [
```

Comment:
i'm not concerned about this being returned by the API unless it's an easy fix. But check my comment in the ticket abotu which codes to use. And let's use a less machine-looking label to make it more readable. i'll provide those translations before we merge.